Source/ASNetworkImageNode.mm

```diff
@@ -719,11 +719,6 @@ - (void)_lazilyLoadImageIfNecessary
           strongSelf->_cacheUUID = nil;
           
           ASPerformBlockOnMainThread(^{
-            __typeof__(self) strongSelf = weakSelf;
-            if (strongSelf == nil) {
-              return;
-            }
-            
```

Comment:
I believe this code should stay – we want to "re-strongify" when we reach the main thread.

Reply:
I can't understand you intention for doing such thing because `strongSelf` is already captured and can't pass away

Reply:
`strongSelf` will be released at the end of the ASPerformBlockOnBackgroundThread block, which may happen before the main thread services our ASPerformBlockOnMainThread call.

Reply:
So its imporrant that the main thread block not capture strongSelf, and instead capture weakSelf and then attempt to retain it

Reply:
Thanks for explanation. I’ve understood your intention. I supposed that it isn’t critical to capture strongSelf in this case because there are only delegate calls inside the block. I will revert my change

Reply:
Cool, thanks.